### ISSUE TITLE
PHP Version 

### ISSUE THREAD

**Issue by DarvinosV** [nielsiano/dmpl-builder#11]
Hello 
CAn i run app without 7.0 php 
dmpl-builder --apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX
Cannot use nielsiano/dmpl-builder's latest version 1.0.1 as it requires php ~7.0 which is not satisfied by your platform.

In PackageDiscoveryTrait.php line 318:

  Package nielsiano/dmpl-builder has requirements incompatible with your PHP version, PHP extensions and Composer version:
    - nielsiano/dmpl-builder 1.0.1 requires php ~7.0 which does not match your installed version 8.2.11.
    
??

**Comment by nielsiano** [comment]
Yes, should run fine. The requirement should be ditched or upgraded, you are free to do so.
